internal/webpagetest.py

```diff
@@ -402,6 +402,10 @@ def get_test(self):
             uptime = self.get_uptime_minutes()
             if uptime is not None:
                 url += '&upminutes={0:d}'.format(uptime)
+            if 'browser_versions' in self.options and \
+                    len(self.options.browser_versions):
+                versions = ','.join(self.options.browser_versions)
+                url += '&browsers={0}'.format(versions)
```

Comment:
It looks like the versions are colon delimited. I'm pretty sure that you need to url encode the versions string to be safe (even if it is working in testing). something like ...format(urllib.quote_plus(versions))

Reply:
Will do, thanks!